horace_core/algorithms/private/gen_sqw_check_files.m

```diff
@@ -221,6 +221,8 @@
 if ~isempty(sqw_file) % we may not want to write a file and return an object instead
     [ok,sqw_exist,sqw_file_out,mess] = check_file_writable(sqw_file,require_sqw_exist);
     if ~ok
+        % replace possible issues with \ in filepath
+        mess = regexprep(mess,'\\','/');
```

Comment:
because current code does not print correct path on Windows. As this is exception is possibly thrown before long operation, the cost is absolutely minisculus and irrelevant. Can be replaced with `replace` but to be fair, absolutely mimisqulus change.